src/components/App.tsx

```diff
@@ -621,6 +621,17 @@ class App extends React.Component<AppProps, AppState> {
                       closable={this.state.toast.closable}
                     />
                   )}
+                  {this.state.contextMenu && (
+                    <ContextMenu
+                      items={this.state.contextMenu.items}
+                      top={this.state.contextMenu.top}
+                      left={this.state.contextMenu.left}
+                      elements={this.scene.getNonDeletedElements()}
+                      appState={this.state}
+                      setAppState={this.setAppState}
```

Comment:
can it use the hooks for app State/ setAppState ?

Reply:
I can see the variable `contextMenuOpen`, can we not remove it since now its controlled via state ?